### ISSUE TITLE
crash

### ISSUE THREAD

**Issue by jkbentley4** [zorgluf/souvenirs-android#20]
Souvenirs crashed when I deleted a page in an album. Thanks for looking at it.
```
type: crash
osVersion: google/raven/raven:14/UQ1A.240105.002/2024011300:user/release-keys
package: fr.nuage.souvenirs:28
process: fr.nuage.souvenirs
processUptime: 112458 + 200 ms
installer: com.android.packageinstaller

java.lang.IllegalArgumentException: Invalid target position
	at androidx.recyclerview.widget.RecyclerView$SmoothScroller.start(RecyclerView.java:11807)
	at androidx.recyclerview.widget.RecyclerView$LayoutManager.startSmoothScroll(RecyclerView.java:8470)
	at fr.nuage.souvenirs.view.helpers.CenterLayoutManager.smoothScrollToPosition(CenterLayoutManager.java:28)
	at androidx.recyclerview.widget.RecyclerView.smoothScrollToPosition(RecyclerView.java:1790)
	at fr.nuage.souvenirs.view.EditPageFragment.lambda$onCreateView$11$fr-nuage-souvenirs-view-EditPageFragment(EditPageFragment.java:240)
	at fr.nuage.souvenirs.view.EditPageFragment$$ExternalSyntheticLambda13.run(Unknown Source:6)
	at android.os.Handler.handleCallback(Handler.java:958)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:205)
	at android.os.Looper.loop(Looper.java:294)
	at android.app.ActivityThread.main(ActivityThread.java:8279)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ExecInit.main(ExecInit.java:49)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:359)
```

**Comment by zorgluf** [comment]
Thanks for the report. Fix it and integrated into v2.8.4 release. This will also correct some random crash on new album creation.

**Comment by jkbentley4** [comment]
Thank you for fixing this so quickly

-------- Original Message --------
